sr_utilities/scripts/sr_utilities/hand_finder.py

```diff
@@ -92,7 +92,7 @@ def get_default_joints(cls):
                   'THJ5', 'WRJ1', 'WRJ2']
         return joints
 
-    def __init__(self, mapping, joint_prefix):
+    def __init__(self, mapping, joint_prefix, hand_h=None):
```

Comment:
OK, I've seen your TODO. So this is supposed to return a list of the joints for a certain hand?
Is it not necessary for hand H then?